docs/guides/user-containers/readme.md

```diff
@@ -6,7 +6,55 @@ applications to be run on Rabbit nodes with access to NNF ephemeral and persiste
 !!! note
 
     The following is a limited look at User Containers.  More content will be
-    provided after the RFC has been finalized.
+    provided after the RFC has been finalized. This is a work in progress.
+
+## Environment Variables
+
+Two sets of environment variables are available with container workflows: Container and Compute Node. The former are the variables that are available inside the user containers. The latter are the variables that are provided back to the DWS workflow, which in turn are collected by Flux and provided to compute nodes. See the Flux documentation for more details.
```

Comment:
I try to avoid calling out Flux, and just say "WLM" instead. Maybe over cautious.

Reply:
Removed